internal/config/config_interface.go

```diff
@@ -0,0 +1,17 @@
+package config
+
+type configClient interface {
+	// global configuration
+	GetConfigurutionData() ([]byte, error)
+	ParseConfiguration() error
+
+	//ebpf engine based on falco configuration
```

Comment:
[Question]: Why is this in the `configClient` interface? You should add a ebpfEngine interface or something like that